user-guide/Advanced_Functionality/Databases/Database_queries/Querying_an_Elasticsearch_database.md

```diff
@@ -101,11 +106,13 @@ To set the replica count to 2 for all indices, use the following query:
 PUT /all/_settings
 {
   "index" :{
-    "number_of_replica" : 2
+    "number_of_replicas" : 2
   }
 }
 ```
 
+Note: For ElasticSearch version 6.8.23 replace PUT /all/_settings with PUT /*/_settings as PUT /all/_settings does not work with ElasticSearch version 6.8.23
+
```

Comment:
I'm not sure here which version does work with PUT /all/_settings since we only support Elastic 6.8.0. Was this tested in the first place? On which version?